appveyor.yml

```diff
@@ -28,9 +29,6 @@
     - md "src\build\nuget"
     - nuget pack "src\nuget\Radical.Windows.Presentation.CastleWindsor.nuspec" -Version "%GitVersion_NuGetVersion%" -Properties "configuration=%CONFIGURATION%" -OutputDirectory "src\build\nuget"
 
-#  pull_requests:
-#    do_not_increment_build_number: true
```

Comment:
ok to remove this as it is useless now because `GitVersion` is renaming the AppVeyor version anyway